rusttorney-server/src/networking/database.rs

```diff
@@ -0,0 +1,59 @@
+use bb8::{Pool, PooledConnection, RunError};
+use bb8_postgres::PostgresConnectionManager;
+use std::net::IpAddr;
+use tokio_postgres::{Error, NoTls};
+
+pub struct DbWrapper {
+    db_pool: Pool<PostgresConnectionManager<NoTls>>,
+}
+
+impl Clone for DbWrapper {
```

Comment:
I think this is exactly what `#[derive(Clone)]` would produce (note for future)

Reply:
I implemented it manually for the sake of clarity that underlying Pool uses Arc inside